lib/libsacd/dstdec/dst_decoder.c

```diff
@@ -75,6 +75,8 @@ static unsigned processor_count(void)
 {
 #if defined(_WIN32)
     return pthread_num_processors_np();
+#elif defined(__ANDROID__)
```

Comment:
Could be changed to the following if the real count improves something
```suggestion
#elif defined(__ANDROID_API__) && __ANDROID_API__ < 23
```

Reply:
nah. dst has been added to ffmpeg, so now the decoder is entirely unused.